SDCTA/src/components/Profile/card_chip.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from "react";
+import Card from "@mui/material/Card";
+import Grid from "@mui/material/Grid";
+import CardMedia from "@mui/material/CardMedia";
+import DeleteIcon from "@mui/icons-material/Delete";
+import IconButton from "@mui/material/IconButton";
+import { Typography } from "@mui/material";
+
+interface CardProps {
+  name: string;
+  cardNumber: string;
+  type: string;
```

Comment:
We probably want an enum for this card type so that it is clear to the client of this component which card types are available. This is especially applicable here since it will likely be another dev using this component. The CardTypes map you have below would then map the enum to the JPG strings. That map is an implementation detail of this component, but since the type is just any string, the user of this component needs to look at that map to see the available card types.